From 7f6cdacf5c5e7183b21f1bd9bc886b920efc7e94 Mon Sep 17 00:00:00 2001
From: Dorin Ionita <dorin.ionita@nxp.com>
Date: Wed, 13 Oct 2021 17:39:27 +0300
Subject: [PATCH 18/20] llce: s32g2: Fixed message to userspace when
 deinitialising LLCE CAN channs.

When deinitialising LLCE CAN channels, the userspace message was wrongly
saying that it was doing the opposite (initialising). This commit fixes
this minor bug.

Issue: ALB-7960
Upstream-Status: Pending 

Signed-off-by: Dorin Ionita <dorin.ionita@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/net/can/llce/llce_can.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/can/llce/llce_can.c b/drivers/net/can/llce/llce_can.c
index 3c1f073353e1..207c171079bf 100644
--- a/drivers/net/can/llce/llce_can.c
+++ b/drivers/net/can/llce/llce_can.c
@@ -147,7 +147,7 @@ static int llce_can_deinit(struct llce_can *llce)
 
 	ret = send_cmd_msg(conf_chan, &cmd);
 	if (ret) {
-		dev_err(dev, "Failed to init LLCE CAN\n");
+		dev_err(dev, "Failed to deinit LLCE CAN\n");
 		return ret;
 	}
 
-- 
2.17.1

